src/ventures-for-shared-prosperity/index.njk

```diff
@@ -7,63 +7,60 @@ title: Ventures for Shared Prosperity
 {% block masthead %}
   {{ super() }}
   <div class="hero hero--centered standard-restricted-width">
+  <img src="" alt="colorful octogon"/>
+  <p>COHORT ONE: 2021-2022 </p>
```

Comment:
All caps text is a visual treatment. It's best to do all caps not in html, but with CSS.

Reply:
Fixed!